### ISSUE TITLE
Notice: Undefined index / Warning: Invalid argument supplied... qoob.php on line 258

### ISSUE THREAD

**Issue by Alecto** [webarkio/wp_qoob#18]
Two errors have recently appeared.
https://image.prntscr.com/image/Ty2z5SYoQ8OFL2S3RUX-vQ.png

I did not notice them before, maybe they appeared after the recent upgrade of WordPress to version 4.8.1
I use PHP 7.1
```
Notice: Undefined index: res in /home/fomenko/subdomains/single/wp-content/plugins/qoob/qoob.php on line 258

Warning: Invalid argument supplied for foreach() in /home/fomenko/subdomains/single/wp-content/plugins/qoob/qoob.php on line 258
```

I tried to connect an empty library according to your example - the same error occurs.
http://webark.com/qoob/how-to/how-to-create-and-add-your-library/

**Comment by jjuja** [comment]
Hi! Thanks for your feedback! We’ve solved this issue in a new Qoob version. We’ll update Qoob shortly. Have a nice day!

**Comment by Alecto** [comment]
thanks for your great work!

**Comment by Alecto** [comment]
I updated the plugin to version 2.0.5
But, the problem was not solved. At the moment it is hidden and there is no error on the page.
But, the Query Monitor shows an error on the Qoob Lib page 
/wp-admin/admin.php?page=qoob-manage-libs

1
Invalid argument supplied for foreach()
wp-content/plugins/qoob/qoob.php:258
Qoob->getLibs()
wp-content/plugins/qoob/views/page-manage-libs.php:2
Qoob->manageLibs()
wp-includes/class-wp-hook.php:298
do_action('qoob_page_qoob-manage-libs')
wp-admin/admin.php:222

2
Undefined index: res
wp-content/plugins/qoob/qoob.php:258
Qoob->getLibs()
wp-content/plugins/qoob/views/page-manage-libs.php:2
Qoob->manageLibs()
wp-includes/class-wp-hook.php:298
do_action('qoob_page_qoob-manage-libs')
wp-admin/admin.php:222

https://image.prntscr.com/image/yC4h654TQt6tka1HEdPoZQ.png

**Comment by kaganvmz** [comment]
This issue will be solved in 2.0.6.  See changelog - https://ru.wordpress.org/plugins/qoob/#developers

**Comment by Alecto** [comment]
fixed!
thank U!!!!!!!!!!!!!!!!!!!!!!!!!
:)
